migration/migration_blackbox_test.go

```diff
@@ -1365,6 +1368,21 @@ func testMigration106RemoveLinkCategoryConcept(t *testing.T) {
 	})
 }
 
+func testMigration107NumberSequencesTable(t *testing.T) {
+	migrateToVersion(t, sqlDB, migrations[:108], 108)
+	require.True(t, dialect.HasTable("number_sequences_table"))
+}
+
+func testMigration108NumberColumnForArea(t *testing.T) {
+	migrateToVersion(t, sqlDB, migrations[:109], 109)
+	require.True(t, dialect.HasColumn("areas", "number"))
+}
+
+func testMigration109NumberColumnForIteration(t *testing.T) {
+	migrateToVersion(t, sqlDB, migrations[:110], 110)
+	require.True(t, dialect.HasColumn("iterations", "number"))
```

Comment:
This test should've failed because of https://github.com/fabric8-services/fabric8-wit/pull/2309/files#r221611164 .
Why didn't it fail?

Reply:
Fixed in c97eb35